src/types/types.ts

```diff
@@ -163,6 +163,7 @@ export interface LayoutService {
   hasLeftDimensions: boolean;
   showTotalsAbove: boolean;
   hasPseudoDimOnLeft: boolean;
+  isAlwaysFullyExpanded: boolean;
```

Comment:
I think you can call this `isFullyExpanded`, since the "always" part seems more like a way of telling Engine that it should be fully expanded.

Reply:
makes sense, I will change